waspc/examples/todoApp/src/client/pages/Main.js

```diff
@@ -1,18 +1,7 @@
-import React from 'react'
-
-import logout from '@wasp/auth/logout.js'
-import Todo from '../Todo'
-import '../Main.css'
+import Todo from "../Todo";
 
 const Main = () => {
-  return (
-    <>
-      <button className='btn btn-blue' onClick={logout}>
-        Logout
-      </button>
-      <Todo />
-    </>
-  )
-}
+    return <Todo />;
+};
```

Comment:
If you want to clean up even further, you can completely remove this file (as it boils down to `export default Todo from "../Todo"`).

Reply:
One of those is a `page` and the other is a `component`, I think it's okay to keep the distinction and have a place in the future to put some content besides the `<Todo />` component.